plugin/src/main/java/io/jenkins/plugins/coverage/metrics/model/Type.java

```diff
@@ -0,0 +1,7 @@
+package io.jenkins.plugins.coverage.metrics.model;
+
+public enum Type {
```

Comment:
Type is to generic as class name in my opinion. The reference to coverage is missing.
Maybe LineCoverageType?
A look at the used [Coverage-Model](https://github.com/jenkinsci/coverage-model) might be also helpful to see what is already implemented and how the model classes are named.